Modules/ITS/src/ITSDecodingErrorCheck.cxx

```diff
@@ -106,6 +104,8 @@ Quality ITSDecodingErrorCheck::check(std::map<std::string, std::shared_ptr<Monit
       }
     }
   }
+  end = std::chrono::high_resolution_clock::now();
+  TIME = std::chrono::duration_cast<std::chrono::seconds>(end - start).count();
```

Comment:
You updated `TIME` variable at the end of `check()` function, as result, you use the old `TIME` value from the previous cycle. 

To solve that, can you try to modify the "if"case from the start of the function to something like that: 

```
if (mTFCount == 0) {
    start = std::chrono::high_resolution_clock::now();
    mTFCount++;
} else{
   end = std::chrono::high_resolution_clock::now();
  TIME = std::chrono::duration_cast<std::chrono::seconds>(end - start).count();
}
```

Reply:
Ok, done!